nginx.conf

```diff
@@ -16,11 +16,9 @@ http {
         
         # windows, Paths in a configuration file must be specified in UNIX-style using forward slashes.
         # root <ABSOLUTE PATH TO dist directory content>;
-        root dist/; # necessary to avoid nginx injecting root into try_files directives
-
-        listen 8080;
```

Comment:
Please update this info in `README.md` also

Reply:
Done